OpenRA.Mods.Common/UtilityCommands/CheckSequenceSprites.cs

```diff
@@ -39,6 +41,8 @@ public void Run(ModData modData, string[] args)
 				foreach (var n in nodes)
 					modData.SpriteSequenceLoader.ParseSequences(modData, ts, sc, n);
 			}
+
+			Environment.Exit(failed ? 1 : 0);
```

Comment:
`0` is the default exit when a program closes so isn't necessary here.
If you encounter an error just exit with `1`.